stan/math/fwd/functor/fvar_finite_diff.hpp

```diff
@@ -0,0 +1,113 @@
+#ifndef STAN_MATH_FWD_FUNCTOR_FVAR_FINITE_DIFF_HPP
+#define STAN_MATH_FWD_FUNCTOR_FVAR_FINITE_DIFF_HPP
+
+#include <stan/math/prim/meta.hpp>
+#include <stan/math/prim/functor/apply_scalar_binary.hpp>
+#include <stan/math/prim/functor/finite_diff_gradient_auto.hpp>
+#include <stan/math/prim/fun/value_of.hpp>
+#include <stan/math/prim/fun/sum.hpp>
+#include <stan/math/prim/fun/serializer.hpp>
+
+namespace stan {
+namespace math {
+namespace internal {
+/**
+ * Helper function for aggregating tangents if the respective input argument
+ * was an fvar<T> type.
+ *
+ * Overload for when the input is not an fvar<T> and no tangents are needed.
+ *
+ * @tparam FuncTangentT Type of tangent calculated by finite-differences
+ * @tparam InputArgT Type of the function input argument
+ * @param tangent Calculated tangent
+ * @param arg Input argument
+ */
+template <typename FuncTangentT, typename InputArgT,
+          require_not_st_fvar<InputArgT>* = nullptr>
+inline constexpr double aggregate_tangent(const FuncTangentT& tangent,
+                                          const InputArgT& arg) {
+  return 0;
+}
+
+/**
+ * Helper function for aggregating tangents if the respective input argument
+ * was an fvar<T> type.
+ *
+ * Overload for when the input is an fvar<T> and its tangent needs to be
+ * aggregated.
+ *
+ * @tparam FuncTangentT Type of tangent calculated by finite-differences
+ * @tparam InputArgT Type of the function input argument
```

Comment:
[minor] we almost excusively use camel case for when something is a template parameter so these could just be `FuncTangent` and `InputArg`